packages/headless/src/features/triggers/triggers-slice-functions.ts

```diff
@@ -0,0 +1,69 @@
+import {Trigger} from '../../api/common/trigger';
```

Comment:
These were simply extracted from the original search triggers slice (see https://github.com/coveo/ui-kit/pull/3898/files#diff-1c85a79538d616be9a043b08cbcddcc8801424d7b2be29b46944e24594d29065).

I added return statements only for testing purposes.